web_favicon/views/res_company.xml

```diff
@@ -5,7 +5,7 @@
         <field name="inherit_id" ref="base.view_company_form" />
         <field name="arch" type="xml">
             <notebook position="inside">
-                <page string="Web Favicon" name="favicon">
+                <page string="Web Favicon" name="favicon" groups="web_favicon.group_edit_favicon">
```

Comment:
If you do what have been asked, this PR would amount to this single line (although fixed), so please squash and leave just 1 commit too.

Thanks!